usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -2838,6 +2855,7 @@ CK_RV ep11tok_init(STDLL_TokData_t * tokdata, CK_SLOT_ID SlotNumber,
             goto error;
     }
 
+#ifndef NO_PKEY
```

Comment:
This also excludes the pthread_mutex_init for raw2key_wrap_blob_mutex which is still needed, even without PKEY.
So I guess there needs to be an #endif after the MSA lines, and a new #ifndef before the pthread_mutex_init for pkey_mutex.

Reply:
OK ... but why not leave the get_msa_level() also for non pkey compiles? So we would add the #ifndef NO_PKEY just before if (!ep11tok_pkey_option_disabled(tokdata)

Reply:
get_msa_level() calls s390_stfle() and this uses inline assembler. So I guess this also wont compile on 32 bit. Furthermore, get_msa_level() is in pkey_utils.c which is excluded for 32 bit builds.

Reply:
ok, done.